lib/posthog/feature_flags.rb

```diff
@@ -119,9 +139,19 @@ def get_feature_flag(key, distinct_id, groups = {}, person_properties = {}, grou
 
       flag_was_locally_evaluated = !response.nil?
 
+      request_id = nil
+
       if !flag_was_locally_evaluated && !only_evaluate_locally
         begin
-          flags = get_feature_variants(distinct_id, groups, person_properties, group_properties, true)
+          decide_data = get_all_flags_and_payloads(distinct_id, groups, person_properties, group_properties, false, true)
+          if !decide_data.key?(:featureFlags)
+            logger.debug "Missing feature flags key: #{decide_data.to_json}"
+            flags = {}
+          else
+            flags =stringify_keys(decide_data[:featureFlags] || {})
```

Comment:
```suggestion
            flags = stringify_keys(decide_data[:featureFlags] || {})
```

Do we have a linter?  Feels like one would catch things like this.